src/main/java/org/coffeemine/app/spring/components/EventsDialog/TaskModification.java

```diff
@@ -24,12 +30,25 @@ public TaskModification(int task_id, Consumer<ITask> callback) {
         final var desc = new TextArea();
         desc.setPlaceholder("Please provide a task description here");
         desc.setValue(task.getDescription());
-        final var sprint_sel = new Select<>("Sprint 1", "Sprint 2", "Sprint 3");
+
+        final var assignees_sel = new MultiselectComboBox<User>();
+        assignees_sel.setItems(NitriteDBProvider.getInstance().getUsers().collect(Collectors.toList()));
+        assignees_sel.setPlaceholder("Assigning to..");
+        Set<User> currentAssignees = new HashSet<>();
+        task.getAssignees().forEach(assigneesId -> currentAssignees.add(NitriteDBProvider.getInstance().getUser(assigneesId)));
+        assignees_sel.setValue(currentAssignees);
+
+        final var sprints = new String[(int) NitriteDBProvider.getInstance().getSprints4Project(NitriteDBProvider.getInstance().getCurrentProject(CurrentUser.get())).count()];
+        for(int i=1; i<= sprints.length; i++){
+            sprints[i-1]= "Sprint " + i;
+        }
```

Comment:
Naked for-loops are 99% of the time a red flag indicating that you're not using abstractions well enough